sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/Subnet.java

```diff
@@ -34,6 +34,13 @@ public interface Subnet extends HasInnerModel<SubnetInner>, ChildResource<Networ
     /** @return the address space prefix, in CIDR notation, assigned to this subnet */
```

Comment:
nit, we may want to write here that e.g. "call `addressPrefixes()` if subnet has multiple prefixes"?

Reply:
updated